interrupt-server.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
```

Comment:
for my understanding, did you run into a particular error that prompted this? Seems like a good default regardless

Reply:
:sweat: the existing image was incompatible with the reverted ruby deps, and failed to run jekyll successfully. in trying to back it down, i ran into the Pillow install issue, and noticed this script would happy continue launching the server even if the docker build step failed :cry: so flipped this on to prevent it